src/Compilers/Server/VBCSCompilerTests/DesktopBuildServerControllerTests.cs

```diff
@@ -6,29 +6,29 @@
 
 namespace Microsoft.CodeAnalysis.CompilerServer.UnitTests
 {
-    public sealed class DesktopBuildServerControllerTests
+    public sealed class BuildServerControllerTests
```

Comment:
>BuildServerControllerTests [](start = 24, length = 26)

Consider renaming the file to match.

Reply:
Will be doing this in a future PR. There are a few renames and I didn't want to rename during the edits for fear it would make it harder to review. 

---
In reply to: [319602914](https://github.com/dotnet/roslyn/pull/38411#discussion_r319602914) [](ancestors = 319602914)